src/main/java/frc/robot/subsystems/ShooterSubsystem.java

```diff
@@ -45,50 +49,45 @@
    * proportional path against the differential and integral paths is controlled
    * by this value.
    */
-  private final double kP = 5e-5;// 0.107;
+  private static final double kP = 5e-5;
   /**
    * The Integral Gain of the SparkMAX PIDF controller The weight of the integral
    * path against the proportional and differential paths is controlled by this
    * value.
    */
-  private final double kI = 1e-6;
+  private static final double kI = 1e-6;
   /**
    * The Differential Gain of the SparkMAX PIDF controller. The weight of the
    * differential path against the proportional and integral paths is controlled
    * by this value.
    */
-  private final double kD = 0;
+  private static final double kD = 0;
   /**
    * The Integral Zone of the SparkMAX PIDF controller. The integral accumulator
    * will reset once it hits this value.
```

Comment:
The integral accumulator will reset if the error exceeds this value.  0 is disabled.

Reply:
The default NEO constants taken from last year's code have the IZ as 0, so that's what I went off of.